src/Components/Header/Header.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
 import Form from "../Form/Form";
+import { Switch, Link, Route } from "react-router-dom";
 import './Header.css'
 
 const Header = () => {
   return (
-    <div className="header">
-      <h1>BookWise</h1>
-      <Form />
-    </div>
+    <Switch>
+      <Route exact path='/'>
+        <div className="header">
+          <h1>BookWise</h1>
+          <Form />
+        </div>
+      </Route>
+      <Route exact path='/:id'>
+        <div className="header">
+          <Link to='/'><h1>BookWise</h1></Link>
```

Comment:
Need to update styling so it doesn't do the purple underlined text when an element is selected. Same goes for all other links.